.github/renovate.json5

```diff
@@ -1,10 +1,26 @@
 {
   $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   ignorePaths: ['**/node_modules/**', '**/implementations/**'],
-  packageRules: [
```

Comment:
we need this package rule to ensure we don't get bump to node 20

Reply:
Apologies @dariuszkuc! I'm not sure how I managed to remove this. Definitely wasn't my goal!